chia/wallet/wallet_interested_store.py

```diff
@@ -30,6 +31,10 @@ async def create(cls, wrapper: DBWrapper2):
             fields = "asset_id text PRIMARY KEY, name text, first_seen_height integer, sender_puzzle_hash text"
             await conn.execute(f"CREATE TABLE IF NOT EXISTS unacknowledged_asset_tokens({fields})")
 
+            # Table for coin states of unknown CATs
+            fields = "coin_state blob PRIMARY KEY, asset_id text, peer blob, fork_height int"
```

Comment:
I'd add an index for `asset_id` since that's primary filter when querying 
and you can save ~50% of space if you make `asset_id` a blob